src/model/claims.rs

```diff
@@ -42,6 +56,198 @@ impl Claims {
     pub fn grant_type(&self) -> &GrantType {
         &self.gty
     }
+
+    #[cfg(test)]
+    pub fn custom_claims(&self) -> &Vec<CustomField> {
+        &&self.custom_claims
+    }
+}
+
+impl<'de> Deserialize<'de> for Claims {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: serde::Deserializer<'de>,
+    {
+        struct ClaimsVisitor;
+
+        enum Field {
```

Comment:
Why this complex? Why not just do something similar to what i've done for `user_info`?
https://github.com/primait/localauth0/blob/master/src/model/user_info.rs#L42

Reply:
Don't get what should this add..

Reply:
for serialization I made the same you did. This complexity is due to deserialization (needed for test "its_possible_to_generate_jwks_and_parse_claims_using_given_jwks_test" )

Reply:
So then you maybe just add that deserialization available only for tests? Or use a raw json string for tests?